subsys/random/rand32_entropy_device.c

```diff
@@ -13,6 +13,52 @@ static const struct device *const entropy_dev =
 	DEVICE_DT_GET(DT_CHOSEN(zephyr_entropy));
 
 #if defined(CONFIG_ENTROPY_DEVICE_RANDOM_GENERATOR)
+uint8_t z_impl_sys_rand8_get(void)
+{
+	uint8_t random_num;
```

Comment:
Would it be simpler to do this here, or is it important to use smaller number of bytes from entropy?
```
return (uint8_t)sys_rand32_get();
```
and similar comment to 16-bit version.

Reply:
there it is important, explicitly for external entropy sources, which are connected via I2C or SPI, resources could be saved. For example the driver of my PR #69920 would benefit from it.